cmd/idminer/main.go

```diff
@@ -176,3 +211,88 @@ func runKeytool() {
 
 	log.Infof("pubkey hex is: %s", hex.EncodeToString(privateKey.PubKey().Serialize()))
 }
+
+func runRPC() {
+	if err := client.Init(configFile, []byte("")); err != nil {
+		log.Fatalf("init rpc client failed: %v", err)
+		os.Exit(-1)
+		return
+	}
+
+	req, resp := resolveRPCEntities()
+
+	// fill the req with request body
+	if err := json.Unmarshal([]byte(rpcReq), req); err != nil {
+		log.Fatalf("decode request body failed: %v", err)
+		os.Exit(-1)
+		return
+	}
+
+	if err := rpc.NewCaller().CallNode(proto.NodeID(rpcEndpoint), rpcName, req, resp); err != nil {
+		// send request failed
+		log.Fatalf("call rpc failed: %v", err)
+		os.Exit(-1)
+		return
+	}
+
+	// print the response
+	if resBytes, err := json.MarshalIndent(resp, "", "  "); err != nil {
+		log.Fatalf("marshal response failed: %v", err)
+		os.Exit(-1)
+		return
+	} else {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
if block ends with a return statement, so drop this else and outdent its block (move short variable declaration to its own line if necessary)